nixos/modules/services/games/factorio.nix

```diff
@@ -120,6 +201,7 @@ in
           "--config=${cfg.configFile}"
           "--port=${toString cfg.port}"
           "--start-server=${mkSavePath cfg.saveName}"
+          "--server-settings=${serverSettingsFile}"
           (optionalString (cfg.mods != []) "--mod-directory=${modDir}")
           (optionalString (cfg.autosave-interval != null) "--autosave-interval ${toString cfg.autosave-interval}")
```

Comment:
`autosave-interval` shouldn't need to go on cmdline if it's in the config file. Does the config file support changing the mod directory as well? If so, that can move there as well.